analysis/filter/EdgeFilter/src/main/kotlin/de/maibornwolff/codecharta/filter/edgefilter/EdgeProjectBuilder.kt

```diff
@@ -9,15 +9,14 @@ import de.maibornwolff.codecharta.model.Path
 import de.maibornwolff.codecharta.model.Project
 import de.maibornwolff.codecharta.model.ProjectBuilder
 import mu.KotlinLogging
-import kotlin.math.max
 
 class EdgeProjectBuilder(private val project: Project, private val pathSeparator: Char) {
     private val logger = KotlinLogging.logger {}
 
     private val projectBuilder = ProjectBuilder(
-        listOf(MutableNode("root", NodeType.Folder)),
-        mutableListOf(),
-        getAttributeTypes()
+            listOf(MutableNode("root", NodeType.Folder)),
```

Comment:
This seems like one identation to much?

Reply:
It seems like the Intellij formatter is a bit confused. Fixed in the next commit 👍